workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,20 +51,35 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
+        {
+            GetClientLogFilename();
+            Launch("Launch standalone client", "local worker launch UnityClient default", "*unityclient.log");
```

Comment:
This is very hard coded to using `UnityClient` and makes an assumption that they have a default config? They all use the same log file too by the look of it.

Reply:
Probably fine for anything we do now. But something to note.